ui/apps/platform/src/Containers/Workflow/WorkflowEntityPageLayout.js

```diff
@@ -88,6 +91,23 @@ const WorkflowEntityPageLayout = ({ location }) => {
 
     const pdfId = pageListType ? 'capture-list' : 'capture-widgets';
 
+    // TODO: remove all this feature flag check after VM updates have been live for one release
+    const showVmUpdates = isFeatureFlagEnabled('ROX_FRONTEND_VM_UDPATES');
+    const useCaseOptions = useCaseEntityMap[useCase].filter((option) => {
+        if (showVmUpdates) {
+            if (option === entityTypes.CVE) {
+                return false;
+            }
+        } else if (
+            option === entityTypes.IMAGE_CVE ||
+            option === entityTypes.NODE_CVE ||
+            option === entityTypes.CLUSTER_CVE
```

Comment:
I think I made changes here as well. You can go with your solution since I think it's cleaner. The only changes you'll need to make is adding a check for `entityTypes.NODE_COMPONENT` and `entityTypes.IMAGE_COMPONENT` here and `entityTypes.COMPONENT` above. Also, adding the `NODE_COMPONENT` and `IMAGE_COMPONENT` to the that entity map